### PR TITLE
Document update procedure of Nexus and SonarQube

### PR DESCRIPTION
**Pull request by michaelsauter**
To make it very easy to ensure a correct local state, repos.sh learns to
deal with the ods-configuration repository.

### CODE REVIEW

File: docs/modules/nexus/pages/administration.adoc [hunk 1]
```diff
@@ -0,0 +1,45 @@
+= Nexus Administration
+
+The `ods-core` repository contains a build recipe for a central Nexus instance. The remainder of this document describes how that setup is done, and how it can be modified and updated.
+
+== Updating Nexus to a new version
+
+The Nexus deployment is using an image built in the central ODS namespace. This image is a customisation of the https://hub.docker.com/r/sonatype/nexus3[official Nexus image on DockerHub].
```
Comment (felipecruz91) [inline docs/modules/nexus/pages/administration.adoc:7]: Where can the reader learn more about the customization mentioned?

File: docs/modules/nexus/pages/administration.adoc [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+The Nexus version which is used is determined by which image tag from DockerHub is used, and depending on tag, at which time the ODS image is built.
```
Comment (felipecruz91) [inline docs/modules/nexus/pages/administration.adoc:9]: Perhaps a link to the Dockerfile indicating which tag is being used at the moment? https://github.com/opendevstack/ods-core/blob/master/nexus/docker/Dockerfile

**Review by stitakis — DISMISSED** [review]

**Review by akhilsoman — APPROVED** [review]
Worked well for us in Nexus upgrade. 👍

File: docs/modules/nexus/pages/administration.adoc (hunk 1 above)
Comment (michaelsauter) [reply docs/modules/nexus/pages/administration.adoc]: Addressed in 2c6cb49.

File: docs/modules/nexus/pages/administration.adoc (hunk 2 above)
Comment (michaelsauter) [reply docs/modules/nexus/pages/administration.adoc:9]: Addressed in 2c6cb49. Note that the tag in the `Dockerfile` does not represent what actually gets built. This is determined by the `BuildConfig`.

**Pull request merged**